### PR TITLE
Remove multiplexing from ShowCollection (use Multiplexer instead)

### PR DESCRIPTION
**Pull request by mmandlis**

### CODE REVIEW

File: runtime/description.js [hunk 1]
```diff
@@ -177,7 +177,7 @@ export class DescriptionFormatter {
     // "A and b."
     // "A, b, ..., and z." (Oxford comma ftw)
     let delim = ['', '', ' and ', ', and '][Math.min(3, count)];
-    return nonEmptyStrings.slice(0, -1).join(', ') + delim + strings.pop();
+    return nonEmptyStrings.slice(0, -1).join(', ') + delim + nonEmptyStrings[count - 1];
```
Comment (sjmiles) [inline runtime/description.js]: Consider
```
const lastString = nonEmptyStrings.pop();
return `${nonEmptyStrings.join(', ')}${delim}${lastString}`;
```

Also, I'm still not a fan of `!!`. Your filter above is semantically equivalent to `(str => str)`, but if you want to cast the return value (for reader clarity?) I recommend `Boolean(str)`.

File: shell/artifacts/Common/Multiplexer.manifest [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 shape HostedParticleShape
   HostedParticleShape(in ~a)
-  consume annotation
+  consume
```
Comment (sjmiles) [inline shell/artifacts/Common/Multiplexer.manifest]: What does it mean to have no consume target?

File: shell/artifacts/Products/ShowProduct.manifest [hunk 3]
```diff
@@ -12,7 +12,5 @@ particle ShowProduct in 'source/ShowProduct.js'
   ShowProduct(in Product product)
   affordance dom
   affordance dom-touch
-  consume root
-    provide annotation
-      view product
-  description `show product`
+  consume item
+  //description `show product`
```
Comment (sjmiles) [inline shell/artifacts/Products/ShowProduct.manifest]: Intended to leave disable?

File: shell/artifacts/Products/source/ShowProduct.js [hunk 4]
```diff
@@ -78,10 +78,9 @@ ${styles}
         <img src="{{resolvedImage}}">
       </div>
     </div>
-    <div slotid="annotation" subid="{{subId}}">
     </div>
   </div>
-  `;
+  `.trim();
```
Comment (sjmiles) [inline shell/artifacts/Products/source/ShowProduct.js]: If you use the `html` template tag (provided by `defineParticle`) you get the `trim()` built-in and a good likelihood your editor can syntax highlight the templates.

File: shell/artifacts/Common/Multiplexer.manifest (hunk 2 above)
Comment (mmandlis) [reply shell/artifacts/Common/Multiplexer.manifest]: in a shape, it means that we don't enforce the name of a consumed slot.

File: runtime/description.js (hunk 1 above)
Comment (mmandlis) [reply runtime/description.js]: done

File: shell/artifacts/Products/ShowProduct.manifest (hunk 3 above)
Comment (mmandlis) [reply shell/artifacts/Products/ShowProduct.manifest]: it is, added a comment and a TODO to uncomment it, when nested recipes are further along.

File: shell/artifacts/Products/source/ShowProduct.js (hunk 4 above)
Comment (mmandlis) [reply shell/artifacts/Products/source/ShowProduct.js]: done

**Pull request merged**